src/lib.rs

```diff
@@ -167,17 +177,22 @@
 //!
 //!
 
+#[allow(unused_imports)]
+#[macro_use]
+extern crate specs_derive;
```

Comment:
I was hoping to allow `specs_derive` to re-export the procedural macro to the user, meaning they don't need to add `extern crate specs_derive;` to their project. If you prefer that people add it explicitly as a separate crate to their projects, I can remove it.